scripts/checks.sh

```diff
@@ -15,8 +15,13 @@ source /etc/profile
 # Set aside arguments, come back to them later
 current_args=( "$@" )
 set --
-# shellcheck disable=SC1091
-source /usr/sbin/resin-vars
+# We still need to include resin-vars on legacy systems
+if [ -f /usr/sbin/resin-vars ]; then
+  # shellcheck disable=SC1091
+  source /usr/sbin/resin-vars
+else
+  source /usr/sbin/balena-config-vars
```

Comment:
you just need to disable the shellcheck directive above here as well

Reply:
@xginn8 Have we stopped using shellcheck?

Reply:
no, but since we run shellcheck on a plain old alpine instance rather than a balenaOS device these files won't exist. this is why the tests are still failing for you.

Reply:
you removed the `disable=`, but what i meant was that you need to add a second `disable=` for the second `source` statement.

Reply:
Ok, got it this time.